code/game/objects/structures/rappel_system.dm

```diff
@@ -230,6 +230,11 @@
 	rope.update_icon_state()
 	flick("rope_deploy", rope)
 	SSminimaps.add_marker(rope, MINIMAP_FLAG_MARINE, image('icons/UI_icons/map_blips.dmi', null, "rappel"))
+	var/area/rappel_area = get_area(target)
+	if(!(rappel_area.area_flags & MARINE_BASE))
+		SEND_GLOBAL_SIGNAL(COMSIG_GLOB_TADPOLE_LANDED_OUT_LZ)
```

Comment:
This signal being sent in 2 places which is against how signals are designed( only sent in one space)